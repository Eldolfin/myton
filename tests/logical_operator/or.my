# Note: These tests implicitly depend on non-zero ints being truthy.

# Return the first True argument.
print 1 or True # expect: 1
print False or 1 # expect: 1
print False or False or True # expect: True

# Return the last argument if all are False.
print False or False # expect: False
print False or False or False # expect: False
print False or False or False or 0 # expect: 0
