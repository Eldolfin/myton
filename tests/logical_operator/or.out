1
1
True
False
False
0
